lib/plugins/aws/package/compile/events/sns.js

```diff
@@ -23,6 +23,7 @@ class AwsCompileSNSEvents {
             topicName: { type: 'string', maxLength: 256, pattern: '^[\\w-]+$' },
             displayName: { type: 'string', minLength: 1 },
             filterPolicy: { type: 'object' },
+            filterPolicyScope: { enum: ['MessageBody', 'MessageAttributes'] },
```

Comment:
Let's maybe simplify it to `{ type: "string" }`, so value is validated on AWS side and we do not have dependency on it in the Framework

Reply:
Updated to string :)